manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0

```

Comment:
This doesn't correspond to any of the ARP Entries added by the installer. Either the primary entry needs to be added under `AppsAndFeaturesEntries` or this needs to be changed